src/components/patient/patientCard/index.tsx

```diff
@@ -21,58 +22,7 @@ const PatientCard = ({ patientId }: Props) => {
   const [patientsData, setPatientsData] = React.useState([] as PatientData[]);
 
   React.useEffect(() => {
-    var data = [
-      {
-        firstName: "Prenom1",
-        lastName: "Nom1",
-        id: "1",
-        age: 5,
-        medicalHistory: "21 trisomy",
-        allergies: "None"
-      },
-      {
-        firstName: "Prenom2",
-        lastName: "Nom2",
-        id: "2",
-        age: 10,
-        medicalHistory: "None",
-        allergies: "Grass, Penicillin"
-      },
-      {
-        firstName: "Prenom3",
-        lastName: "Nom3",
-        id: "3",
-        age: 56,
-        medicalHistory: "None",
-        allergies: "None"
-      },
-      {
-        firstName: "Prenom4",
-        lastName: "Nom4",
-        id: "4",
-        age: 36,
-        medicalHistory: "Wrist fracture",
-        allergies: "None"
-      },
-      {
-        firstName: "Prenom5",
-        lastName: "Nom5",
-        id: "5",
-        age: 17,
-        medicalHistory:
-          "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum",
-        allergies:
-          "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum, Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum"
-      },
-      {
-        firstName: "Prenom6",
-        lastName: "Nom6",
-        id: "6",
-        age: 96,
-        medicalHistory: "Type 2 diabetes, Stroke",
-        allergies: "None"
-      }
-    ];
+    var data = DATA_TEST;
```

Comment:
why do you need this line ?
also, try to _never_ use the `var` keyword, use `const` instead (more optimized)